v2/apiserver/internal/api/rest/token_auth_filter.go

```diff
@@ -15,6 +15,15 @@ import (
 	"github.com/pkg/errors"
 )
 
+var (
+	// Root is a singleton that represents Brigade's "root" user.
+	root = &api.RootPrincipal{}
+	// Scheduler is a singleton that represents Brigade's scheduler component.
+	scheduler = &api.SchedulerPrincipal{}
+	// Observer is a singleton that represents Brigade's observer component.
+	observer = &api.ObserverPrincipal{}
+)
+
```

Comment:
These are not new. They just moved here from `v2/apiserver/internal/api/rest/named_principals.go` when other elements of that file were relocated to the `api` package.